tests/Integration/Framework/GacelaTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+declare(strict_types=1);
+
+namespace GacelaTest\Integration\Framework;
+
+use Gacela\Framework\Bootstrap\GacelaConfig;
+use Gacela\Framework\Gacela;
+use PHPUnit\Framework\TestCase;
+use ReflectionClass;
+
+final class GacelaTest extends TestCase
+{
+    public function test_null_get_cache_dir(): void
+    {
+        // Needed in order to set up the private `Gacela::$appRootDir as null` for this use case
+        // when you try to access it before bootstrapping the application.
+        $reflectionClass = new ReflectionClass(Gacela::class);
+        $reflectionProperty = $reflectionClass->getProperty('appRootDir');
+        $reflectionProperty->setAccessible(true);
+        $reflectionProperty->setValue(null);
+
+        self::assertNull(Gacela::rootDir());
```

Comment:
Can't you do the following test?
```php
# test A
self::assertNull(Gacela::rootDir());

# test B
Gacela::bootstrap(__DIR__);

self::assertSame(__DIR__, Gacela::rootDir());
```

Reply:
This is exactly what I did in these two tests. Or what do you mean?